install_config/upgrading/manual_upgrades.adoc

```diff
@@ -744,6 +974,119 @@ IMAGE_VERSION=<tag>
 +
 Replace `<tag>` with `{latest-tag}` for the latest version.
 
+. Configure Dnsmasq. Because of changes made in {product-title} 3.6, you will need to
+perform the following steps to configure dnsmasq as part of the upgrade. See DNS Changes
+under xref:../../release_notes/ocp_3_6_release_notes.adoc#ocp-36-notable-technical-changes[Notable Technical Changes]
+in the {product-title} 3.6 Release Notes.
++
+.. Download the following script:
++
+----
+# wget https://raw.githubusercontent.com/openshift/openshift-ansible/release-3.6/roles/openshift_node_dnsmasq/files/networkmanager/99-origin-dns.sh -O /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
+# chmod 755 /etc/NetworkManager/dispatcher.d/99-origin-dns.sh
+----
++
+This command downloads the *99-origin-dns.sh* script and copies the script to the *_/etc/NetworkManager/dispatcher.d/_* directory.
+This script configures pods to use the node IP address as resolver. Using 127.0.0.1 inside a pod would fail.
+
++
+.. Create a *_/etc/origin/node/node-dnsmasq.conf_*  node configuration file.
```

Comment:
Note, `node-dnsmasq.conf` needs to be created twice. Once in `/etc/origin/node/node-dnsmasq.conf` and one time in `/etc/dnsmasq.d/node-dnsmasq.conf`. The file from `/etc/origin/node/node-dnsmasq.conf` will be copied to `/etc/dnsmasq.d/node-dnsmasq.conf` but should still be created as we have found certain issues in case it was not there. I therefore suggest to adjust the doc in a way that we make sure the file is created twice respectively the customer understands that it needs to be in place twice.